src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/TextBufferExtensions.cs

```diff
@@ -2,7 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using Microsoft.VisualStudio.LanguageServerClient.Razor;
+using Microsoft.VisualStudio.LanguageServer.ContainedLanguage;
 
 namespace Microsoft.VisualStudio.Text
 {
```

Comment:
Below here there's a `__Razor_HostDocumentVersionMarker__`. We should probably remove the Razor part

Reply:
Sounds good. I'll just remove the Razor from the string then.

---
In reply to: [437130240](https://github.com/dotnet/aspnetcore-tooling/pull/1994#discussion_r437130240) [](ancestors = 437130240)

Reply:
On the second thought, replaced Razor with MsLsp to be more inline with what we do for our custom lsp messages. Let me know if you would rather use something else or remove that part. 

---
In reply to: [437744237](https://github.com/dotnet/aspnetcore-tooling/pull/1994#discussion_r437744237) [](ancestors = 437744237,437130240)